### ISSUE TITLE
Job submission hangs or fails

### ISSUE THREAD

**Issue by ermsdev** [ASWWU-Web/jobs#22]
On submitting a new job the submit button says "submitting..." indefinitely (in production), and may produce a window.alert error pop up with `[object Object]` in development.

**Comment by ermsdev** [comment]
This is not an issue for some on the live server, but is for others.
I noticed an error on my local machine where the python server returns invalid data for the request to `/forms/application/view/1/<username>`.

For me it returns 
```
{"application": {
    "username": "Stephen.Ermshar", 
    "status": "new", 
    "resume": null, 
    "answers": [
        {"answer": "(test) stephen", "questionID": "1"}, 
        {"answer": "ermshar", "questionID": "2"}, 
        ...
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        {"answer": "How many hours do you hope to work? A", "questionID": "13"},
        ...
]}}
```
The length of the answers array is 80 elements long, the only repeated question is this one with ID 13.

UPDATE: This appears to be an issue with my local testing database. Somehow lots of duplicate answers got entered for the same questionID and applicationID. I've never entered these manually or interacted with the database manually to enter answers, so I'm guessing there's an issue with how the server records answers. maybe it doesn't properly check whether an answer already exists and needs to be updated, it just adds a new answer, I'm not sure why that would happen for only this one answer though.

**Comment by ermsdev** [comment]
Running my local environment on a copy of the production database got rid of weird errors, but I did find out that this doesn't hang specifically when a resume is included. If a resume is not included in the application then the application is still successfully submitted, but the user doesn't receive confirmation in the UI. Currently the user does not receive a confirmation email regardless, but the form owner does get notified.

**Comment by ermsdev** [comment]
fix in #24 by navigating to done page immediately if an applicant has not included a resume.

**Comment by ermsdev** [comment]
deployed under PR #25
